.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,7 @@
 - [ ] All tests are passing.
 - [ ] New or changed API methods have been documented.
 - [ ] `npm run format` has been run
+
+## CHANGELOG
+
```

Comment:
Please update it to reflect current change

Reply:
This is the template!